modules/appeals_api/spec/serializers/evidence_submission_serializer_spec.rb

```diff
@@ -31,24 +25,44 @@
     expect(rendered_hash[:appeal_id]).to eq evidence_submission.supportable.id
   end
 
-  it 'includes :code' do
-    expect(rendered_hash[:code]).to eq evidence_submission.code
+  it 'includes :updated_at' do
+    expect(rendered_hash[:updated_at]).to eq evidence_submission.updated_at
   end
 
-  it "truncates :detail value if longer than #{described_class::MAX_DETAIL_DISPLAY_LENGTH}" do
-    max_length_plus_ellipses = described_class::MAX_DETAIL_DISPLAY_LENGTH + 3
-    expect(rendered_hash[:detail].length).to eq(max_length_plus_ellipses)
+  it 'does not include any extra attributes' do
```

Comment:
Hmmm.. I'm not sure how I feel about this test. On the one hand, it lets us know when anything changes, but it's also not really handling any code path.